front/app/containers/Admin/reporting/containers/ReportBuilderPage/index.tsx

```diff
@@ -60,15 +66,27 @@ const ReportBuilderPage = () => {
               <FormattedMessage {...messages.createReportDescription} />
             </Text>
             <Box display="flex">
-              <Button
-                onClick={openModal}
-                width="auto"
-                mt="12px"
-                bgColor={colors.primary}
-                p="8px 12px"
+              <Tippy
+                maxWidth="250px"
+                placement="right-start"
+                content={
+                  <FormattedMessage {...sharedMessages.contactToAccess} />
+                }
+                disabled={isReportBuilderAllowed}
```

Comment:
```suggestion
                disabled={!isReportBuilderAllowed}
```

This seems a mistake. Should be disabled if not allowed (by pricing plan), right?

Reply:
No it is not, we are disabling the Tippy component in this case

Reply:
Aah, I thought it was about the button. Should have rendered to component. My bad 🙃 In that case the button needs the disabled prop. I can click it when allowed = false.